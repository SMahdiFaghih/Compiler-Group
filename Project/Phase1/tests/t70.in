103.22E-3