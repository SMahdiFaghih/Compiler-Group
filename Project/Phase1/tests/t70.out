T_DOUBLELITERAL 103.22E-3